cloudformation/media-atom-maker-dev.yml

```diff
@@ -30,6 +34,7 @@ Resources:
     Type: "AWS::IAM::AccessKey"
     Properties:
       UserName: {"Ref": "MediaAtomUser"}
+      Serial: 2
```

Comment:
I'm not crazy about putting the access and secret keys into cloudformation output, where they're visible to all. Is this something all the devs share? Be good to have a quick chat IRL about this.

It looks like this only grants very limited access to a dev bucket, so not the end of the world and I understand this is because Janus credentials can't be used in this example.